niworkflows/data/__init__.py

```diff
@@ -8,20 +8,8 @@
 from .getters import (
     get_brainweb_1mm_normal,
     get_ds003_downsampled,
-    get_mni_template,
-    get_mni_template_ras,
-    get_ants_oasis_template,
-    get_ants_oasis_template_ras,
-    get_mni_epi,
-    get_mni152_nlin_sym_las,
-    get_mni152_nlin_sym_ras,
-    get_mni_icbm152_linear,
-    get_mni_icbm152_nlin_asym_09c,
-    get_oasis_dkt31_mni152,
-    get_hcp32k_files,
-    get_conte69_mesh,
```

Comment:
Should we go through a deprecation cycle for these? Or is niworkflows more of a makes-no-promises sort of library?

Reply:
Given that this was a pretty unscalable approach and that fmriprep is the only client of niworkflows... I'd go with makes-no-promises. That will also push people to update fmriprep if they are not using containers, which is positive in the long term.

We should consider always setting a maximum version for niworkflows in fmriprep, though.